aws/eip.go

```diff
@@ -131,8 +133,19 @@ func nukeAllEIPAddresses(session *session.Session, allocationIds []*string) erro
 			if awsErr, isAwsErr := err.(awserr.Error); isAwsErr && awsErr.Code() == "AuthFailure" {
 				// TODO: Figure out why we get an AuthFailure
 				logging.Logger.Debugf("EIP %s can't be deleted, it is still attached to an active resource", *allocationID)
+				telemetry.TrackEvent(commonTelemetry.EventContext{
+					EventName: "Error Nuking EIP",
+				}, map[string]interface{}{
+					"region": *session.Config.Region,
+					"reason": "Still Attached to an Active Resource",
```

Comment:
Very nice. Seems like a subtle thing to have one more field but being able to key off the reason could be a big deal.